crates/zizmor/src/fix.rs

```diff
@@ -59,6 +63,11 @@ pub fn apply_fixes(results: &FindingRegistry, registry: &InputRegistry) -> Resul
         // First, try to apply each fix independently to the original content
         // to collect which fixes can be applied successfully
         for (ident, finding, fix) in fixes {
+            // Skip fixes that are not applicable to the current persona
+            if fix.persona <= persona {
```

Comment:
@woodruffw Do you think this is a good way to do this, considering that we are comparing integers (enum fields)?